src/less/variables/ds6/button-variables.less

```diff
@@ -16,3 +16,10 @@
 @button-secondary-disabled-background-color: @color-white;
 @button-secondary-disabled-border-color: @color-disabled;
 @button-secondary-disabled-foreground-color: @color-disabled;
+
+@button-tertiary-background-color: @color-grey1;
```

Comment:
Given that ds4 and ds6 component tokens have the same product aliases, we can move them out of ds4 and ds6 into base (p.s. I will handle any changes to this PR while Austin is out).